src/dbally/llm_client/base.py

```diff
@@ -80,9 +80,10 @@ async def call(
         Calls LLM API endpoint.
 
         Args:
-            prompt: Text to be asked.
-            response_format: Optional argument used in the OpenAI API - used to force json output
+            prompt: prompt passed to the LLM.
+            response_format: Optional argument used in the OpenAI API - used to force a json output
             options: Additional settings used by LLM.
+            event: an LLMEvent instance which fields should be filled during the method executionOptio.
```

Comment:
I believe the "Optio" part is here by mistake

Reply:
I just tried to invent a new language, unsuccessfully :(